app/view/fund_transfer.py

```diff
@@ -0,0 +1,152 @@
+
+    def get(self):
+
+        """Provides the data of all the fund transfer"""
+        try:
+            all_fund_transfer = FundTransfer.query.all()
+            output = []
+            for transfer in all_fund_transfer:
+                currentfund = {}
+                currentfund['from_account'] = transfer.from_account
+                currentfund['to_account'] = transfer.to_account
+                output.append(currentfund)
+            logger.info("All fund transfer data returned successfully")
+            response = ResponseGenerator(data=output, message="All fund transfer data returned successfully",
+                                         success=True, status=status.HTTP_200_OK)
+            return response.success_response()
+        except Exception as error:
+            logger.exception(error)
+            response = ResponseGenerator(data={}, message="Invalid request", success=False,
+                                         status=status.HTTP_404_NOT_FOUND)
+            return response.error_response()
+
+
+class FundTransferData(Resource):
+
+    """ for FundTransferData GET(single fund transfer detail),
+        PUT(update fund transfer), DELETE(delete fund transfer)"""
+
+    def get(self, id):
+
+        """Gives the data of single fund transfer  with selected fund transfer id """
+
+        try:
+            fund = FundTransfer.query.filter(FundTransfer.id == id).first()
+            output = fund_transfer_schema.dump(fund)
+            if fund:
+                logger.info('fund transfer returned successfully')
+                response = ResponseGenerator(data=output, message="fund transfer returned successfully",
+                                             success=True, status=status.HTTP_200_OK)
+                return response.success_response()
+            else:
+                logger.exception("fund transfer id not found")
+                response = ResponseGenerator(data={}, message="fund transfer id not found", success=False,
+                                             status=status.HTTP_404_NOT_FOUND)
+                return response.error_response()
+        except Exception as error:
+            logger.exception(error)
+            response = ResponseGenerator(data={}, message="fund transfer id not found", success=False,
+                                         status=status.HTTP_404_NOT_FOUND)
+            return response.error_response()
+
+    def put(self, id):
+
+        """Update the fund transfer """
+
+        try:
+            data = request.get_json()
+            result = fund_transfer_schema.validate(data)
+            if result:
+                logger.exception(result)
+                response = ResponseGenerator(data={}, message=result,
+                                             success=False, status=status.HTTP_400_BAD_REQUEST)
+                return response.error_response()
+            fund = FundTransfer.query.filter(FundTransfer.id == id).first()
+            if fund:
```

Comment:
I do not see else condition handled for the two `if` statements.

Reply:
have removed the one if condition because it was not required